openff/toolkit/tests/test_forcefield.py

```diff
@@ -3825,18 +3823,18 @@ def test_freesolv_gbsa_energies(
         amber_energy = get_context_potential_energy(amber_context, positions)
         off_energy = get_context_potential_energy(off_context, positions)
 
```

Comment:
I am not a very clever person so I'd like to keep this in for when this problem comes back to our attention.

```suggestion
        # Very handy for debugging
        # print(openmm.XmlSerializer.serialize(off_gbsa_force))
        # print(openmm.XmlSerializer.serialize(amber_gbsa_force))
```